Example battery config in doc's is wrong

#### Current Behavior
<!-- A clear and concise description of the behavior. -->
Error's with:
[ERROR] - (starship::config): Unable to parse the config file: unexpected character found: `\u{1f4a6}` at line 18 column 22
#### Expected Behavior
<!-- A clear and concise description of what you expected to happen. -->
~ via 🐹 v1.17.2 via  v14.17.2 via 🐘 via 💎 v2.7.3
💦81% ➜
#### Additional context/Screenshots
<!-- Add any other context about the problem here. If applicable, add screenshots to help explain. -->
Error returned:
[ERROR] - (starship::config): Unable to parse the config file: unexpected character found: `\u{1f4a6}` at line 18 column 22
#### Possible Solution
<!--- Only if you have suggestions on a fix for the bug -->
In doc file , change the example code. character 💦is not in double quotes("")
#### Environment
- Starship version: 0.58.0
- nu version: 0.39.0
 #  │      Column0       │                  Column1                  
────┼────────────────────┼───────────────────────────────────────────
  0 │ version            │ 0.39.0                                    
  1 │ short_commit       │ 2b06ce27                                  
  2 │ commit_hash        │ 2b06ce27d3b7d9eec65686586f1c469f908dc7f2  
  3 │ commit_date        │ 2021-10-26 19:36:41 +00:00                
  4 │ build_os           │ windows-x86_64                            
  5 │ rust_version       │ rustc 1.56.0 (09c42c458 2021-10-18)       
  6 │ rust_channel       │ stable-x86_64-pc-windows-msvc (default)   
  7 │ cargo_version      │ cargo 1.56.0 (4ed5d137b 2021-10-04)       
  8 │ pkg_version        │ 0.39.0                                    
  9 │ build_time         │ 2021-10-26 20:04:24 +00:00                
 10 │ build_rust_channel │ release                                   
 11 │ features           │ clipboard-cli, ctrlc, dataframe, default, 
    │                    │ rustyline, term, trash, uuid, which, zip  
 12 │ installed_plugins  │                                           
────┴────────────────────┴───────────────────────────────────────────
- Operating system: Windows 10.0.22000
- Terminal emulator: windows terminal <unknown version>
- Git Commit Hash: 0a60d1e9
- Branch/Tag: v0.58.0
- Rust Version: rustc 1.55.0 (c8dfcfe04 2021-09-06)
- Rust channel: stable-x86_64-pc-windows-msvc (default) release
- Build Time: 2021-09-21 13:43:39 +00:00
#### Relevant Shell Configuration
nushell config:
```nu
startup = [
 "mkdir ~/.cache/starship",
 "starship init nu | save ~/.cache/starship/init.nu",
 "source ~/.cache/starship/init.nu"
]
prompt = "starship_prompt"
```

#### Starship Configuration

```toml
# Inserts a blank line between shell prompts
add_newline = true

# Replace the "❯" symbol in the prompt with "➜"
[character]                            # The name of the module we are configuring is "character"
success_symbol = "[➜](bold green)"     # The "success_symbol" segment is being set to "➜" with the color "bold green"
[battery]
full_symbol = "🔋 "
charging_symbol = "⚡️ "
discharging_symbol = "💀 "
[[battery.display]]  # "bold red" style and discharging_symbol when capacity is between 0% and 10%
threshold = 10
style = "bold red"

[[battery.display]]  # "bold yellow" style and 💦 symbol when capacity is between 10% and 30%
threshold = 100
style = "bold yellow"
discharging_symbol = 💦

```

--- a/docs/config/README.md
+++ b/docs/config/README.md
@@ -406,7 +406,7 @@ style = "bold red"
 [[battery.display]]  # "bold yellow" style and 💦 symbol when capacity is between 10% and 30%
 threshold = 30
 style = "bold yellow"
-discharging_symbol = 💦
+discharging_symbol = "💦"
 
 # when capacity is over 30%, the battery indicator will not be displayed
 

